### PR TITLE
[#168684752] Removed patch react-native-qrcode-scanner

### PR DESCRIPTION
**Pull request by Alex193a**
Removed the patch that fixes the permissions bug on Android < 23 and updated the library to the latest version which includes the fix.

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#168684752](https://www.pivotaltracker.com/story/show/168684752): Rimuovere patch libreria react-native-qrcode-scanner

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/__mocks__/react-native-camera.ts [hunk 1]
```diff
@@ -2,6 +2,23 @@
  * A mocked version of the react-native-camera components.
  */
 
-const Camera = {};
+import React from "react";
```
[the remark this answers is not in the document]
Comment (Alex193a) [reply ts/__mocks__/react-native-camera.ts]: It is used by a test suit. Without this change the _yarn test_ command fails.

[the remark this answers is not in the document]
Comment (Undermaken) [reply ts/__mocks__/react-native-camera.ts]: @Alex193a Are you sure? Just right now I removed that file and all tests run correctly. Also by running only **saveNavigationStateSaga.test.ts** all tests are ok

Could you report the error you get by removing that file?

[the remark this answers is not in the document]
Comment (Alex193a) [reply ts/__mocks__/react-native-camera.ts]:
```
FAIL  ts/sagas/startup/__tests__/saveNavigationStateSaga.test.ts
  • Test suite failed to run

    TypeError: Cannot read property 'Constants' of undefined

      at Object.Constants (node_modules/react-native-qrcode-scanner/index.js:23:34)
      at Object.<anonymous> (ts/screens/wallet/payment/ScanQrCodeScreen.tsx:1568:49)
```

[the remark this answers is not in the document]
Comment (Undermaken) [reply ts/__mocks__/react-native-camera.ts]: In **saveNavigationStateSaga.test.ts** there is no reference to Camera or Constants
Anyway I'm pretty sure this file is no longer used by no one https://github.com/teamdigitale/io-app/commit/200c8aaee356619d3356b4f32b119c555a32e4c3#diff-e3d22ec86d72a4cdaff054a880fe7c4b (it was created as a mock for a test)

Could you commit the file deletion so we can check if the CI fails? Perhaps my project could is still using an old cache of packages

[the remark this answers is not in the document]
Comment (Alex193a) [reply ts/__mocks__/react-native-camera.ts]: Ok, I deleted it and the _yarn test_ command doesn't fail.

**Pull request merged**